extensions/kafka/src/test/java/com/hazelcast/jet/kafka/impl/KafkaTestSupport.java

```diff
@@ -64,12 +64,22 @@ public abstract class KafkaTestSupport {
 
     public static KafkaTestSupport create() {
         if (!dockerEnabled()) {
-            if (System.getProperties().containsKey("test.kafka.version")) {
-                throw new IllegalArgumentException("'test.kafka.version' system property requires docker enabled");
-            }
+            assertPropertyNotSet("test.kafka.version");
+            assertPropertyNotSet("test.redpanda.version");
+            assertPropertyNotSet("test.use.redpanda");
             return new EmbeddedKafkaTestSupport();
         } else {
-            return new DockerizedKafkaTestSupport();
+            if (System.getProperties().containsKey("test.use.redpanda")) {
```

Comment:
Maybe the property should be `test.kafka.use.redpanda` to make it clear that for Kafka connector tests we will use Redpanda. Wdyt?

Reply:
OK, fixed: https://github.com/hazelcast/hazelcast/pull/24599/commits/19a4f84bd4c2c5c3c3c867df0489a61aaed7f63c